atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/async/CqlClientImpl.java

```diff
@@ -108,6 +113,11 @@ public void close() {
         cluster.close();
     }
 
+    @Override
+    public boolean isValid() {
+        return session.isClosed();
+    }
```

Comment:
This seems suspicious - did we intend the other way around? See e.g. [Datastax docs](https://docs.datastax.com/en/drivers/java/2.0/com/datastax/driver/core/Session.html#isClosed--)